### PR TITLE
CDS-3 Update navbar styling

### PR DESCRIPTION
**Pull request by forkball**
Changlog:
- Add logo to nav
- Update styles

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
-  );
+    return (
+        <header className="z-20 fixed flex flex-col w-full">
+            <nav className="relative flex w-full p-6">
+                <a href="/" className="w-1/3 sm:w-24 mr-4">
+                    <img src={logo_image} alt="logo"></img>
+                </a>
+                <div id="nav-content" className="flex items-center">
+                    <h2 className="pr-3">Clarity Detailing Solutions</h2>
+                    <span className="h-full"></span>
```
Comment (Liannus) [inline src/components/Navbar/Navbar.jsx]: I think maybe this should be around h-50% or so, looks a lot better than 100%.

**Review by Liannus — CHANGES_REQUESTED** [review]
One small thing.

Also should we put in the similar functionality to TBW in here for it to go grey/black on scroll.

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (forkball) [reply src/components/Navbar/Navbar.jsx]: Resolved.

**Comment by forkball** [comment]
> One small thing.
> 
> Also should we put in the similar functionality to TBW in here for it to go grey/black on scroll.

Will do in a separate PR, also needs to be responsive for mobile

**Comment by Liannus** [comment]
Sounds good to me.

**Review by Liannus — APPROVED** [review]

**Pull request merged**